### PR TITLE
[PROJECTX-7383] Eastwood Performance tools baseline

### PR DESCRIPTION
**Pull request by fpiagent**
In this PR:
- Eastwood existing features tests and base structure
- Script to run performance in a remote worker and extract results

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,27 @@
-# Alexander performance tests
-## a.k.a Alexander the great
-Here we can found the framework and definition for every performance test suite defined
+# Alexander Performance Tests
```
Comment (pablokraan) [inline README.md]: Why alexander? I would change the name of this....

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+baseUrl=http://18.209.28.58:30000
```
Comment (pablokraan) [inline src/main/resources/application.properties]: Is this IP the real one?

File: src/main/scala/eastwood/Base.scala [hunk 3]
```diff
@@ -0,0 +1,62 @@
+package eastwood
+
+import com.typesafe.config.ConfigFactory
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+
+class Base extends Simulation {
```
Comment (pablokraan) [inline src/main/scala/eastwood/Base.scala]: Would be good to have some "scaladocs" here

File: src/main/scala/eastwood/Base.scala [hunk 4]
```diff
@@ -0,0 +1,62 @@
+package eastwood
```
Comment (pablokraan) [inline src/main/scala/eastwood/Base.scala]: What about license headers?

File: README.md (hunk 1 above)
Comment (fpiagent) [reply README.md]: Updated to `# Mozart Performance Tests`. Keep in mind this was created to perf-test alexander and then repurposed as a "Let's test all of mozart" tool. Needs lots of love and a VISION of what we want from this.

File: src/main/resources/application.properties (hunk 2 above)
Comment (fpiagent) [reply src/main/resources/application.properties]: Set a fake one and renamed to EastwooUrl

File: src/main/scala/eastwood/Base.scala (hunk 3 above)
Comment (fpiagent) [reply src/main/scala/eastwood/Base.scala]: Done

File: src/main/scala/eastwood/Base.scala (hunk 4 above)
Comment (fpiagent) [reply src/main/scala/eastwood/Base.scala]: Added

File: src/main/resources/application.properties (hunk 2 above)
Comment (pablokraan) [reply src/main/resources/application.properties]: I was not concerned about security... just to knwo what parameter must be configured for a load test. Can you document this?

**Pull request merged**